lib/buy/meld/meld_buy_provider.dart

```diff
@@ -155,8 +155,8 @@ class MeldBuyProvider extends BuyProvider {
       'countryCode': countryCode,
       //'paymentMethodType': normalizePaymentMethod(paymentMethod.paymentMethodType),
       'sourceAmount': amount.toString(),
-      'sourceCurrencyCode': quote.sourceCurrency,
-      'destinationCurrencyCode': quote.destinationCurrency,
+      'sourceCurrencyCode': quote.fiatCurrency,
+      'destinationCurrencyCode': quote.cryptoCurrency,
```

Comment:
same

Reply:
According to their docs, it should work as is, and it was working

sourceCurrencyCode -- The type of fiat currency used to purchase cryptocurrency; if not passed in, what shows is the default fiat currency of the country selected

destinationCurrencyCode -- The cryptocurrency to be purchased; if not passed in, the default is BTC